app/scripts/views/base.js

```diff
@@ -615,6 +615,15 @@ define(function (require, exports, module) {
     },
 
     /**
+     * Log an event once per page load
+     *
+     * @param {String} eventName
+     */
+    logEventOnce: function (eventName) {
+      this.metrics.logEventOnce(eventName);
+    },
```

Comment:
I really like this helper, will reduce a lot of confusion when reviewing the metrics.